SYCL/Reduction/reduction_range_2d_rw.cpp

```diff
@@ -48,6 +53,22 @@ int main() {
 
   tests<class C1>(Q, CustomVec<long long>(0), CustomVec<long long>(99),
                   CustomVecPlus<long long>{}, range<2>{33, MaxWGSize});
+  tests<class C2>(Q, CustomVec<long long>(99), CustomVecPlus<long long>{},
```

Comment:
Do we really need to keep the one *with* the identity param? Especially given the new test cases below to test that path.

Reply:
I think it makes sense to test for custom types to make sure that runtime identities also work correctly for custom types.